src/librustc/mir/interpret/error.rs

```diff
@@ -281,6 +284,29 @@ pub enum InterpError<'tcx, O> {
         required: Align,
         has: Align,
     },
+    MemoryLockViolation {
```

Comment:
these suddenly showed up in this PR

Reply:
This was taken from the [RFC](https://gist.github.com/TheDarkula/d5993ff5ec66338ad6063db044068f9f#file-error-rs-L41)